README.md

```diff
@@ -4,3 +4,62 @@
 
 This repository provides support for applications using the [Google Java Client Libraries](https://github.com/googleapis/google-cloud-java)
 to be built as [GraalVM Native Images](https://www.graalvm.org/docs/reference-manual/native-image).
```

Comment:
The link seems to have changed to: https://www.graalvm.org/reference-manual/native-image/.

Reply:
Done.